workloads/network-perf-v2/run.sh

```diff
@@ -15,9 +15,20 @@ log "###############################################"
 log "Workload: ${WORKLOAD}"
 log "###############################################"
 
-timeout $TEST_TIMEOUT ./k8s-netperf --debug --metrics --all --config ${WORKLOAD} --search $ES_SERVER --tcp-tolerance ${TOLERANCE} --clean=true
-run=$?
-
+echo $WORKLOAD
+        if [ $WORKLOAD == iperf ];then
+          timeout $TEST_TIMEOUT ./k8s-netperf --debug --metrics --all --iperf --search $ES_SERVER --tcp-tolerance ${TOLERANCE} --clean=true
```

Comment:
I would introduce new env vars for iperf/local/across if you want to have different use cases.